tests/test_ingest_pipeline.py

```diff
@@ -160,7 +160,9 @@ def test_ingest_no_transform_module(tmpdir):
         os.remove(tm)
 
     # Remove transform module from ingest config
-    ingest_config_path = os.path.join(new_study_dir, "ingest_package_config.py")
+    ingest_config_path = os.path.join(
+        new_study_dir, "ingest_package_config.py"
+    )
```

Comment:
Why did this get reshaped?

Reply:
anyway, run black -l80 on this file with the new version of black.

Reply:
it shouldn't have been. Format on save wanted 79 characters instead of 80 🤦 . Updated my settings, and fixed this.